ocaml/parsing/pprintast.ml

```diff
@@ -1637,7 +1637,9 @@ and module_expr ctxt f x =
   if x.pmod_attributes <> [] then
     pp f "((%a)%a)" (module_expr ctxt) {x with pmod_attributes=[]}
       (attributes ctxt) x.pmod_attributes
-  else match x.pmod_desc with
+  else match Jane_syntax.Module_expr.of_ast x with
+    | Some ext -> extension_module_expr ctxt f ext
+    | None -> match x.pmod_desc with
```

Comment:
Could you add a test to `pprintast_unconditional.ml`? In particular, the print should contain `[@jane.non_erasable.instancee]` because pprintast is supposed to roundtrip with parser.